### DISCUSSION TITLE
Can NOT open output file in Paraview

### DISCUSSION THREAD

**Discussion by seongwonHam** [idaholab/moose#22116] [Q&A Tools] [answered]
Hello, MOOSE users.  

I have some trouble with the ParaView program. 
I can NOT open my MOOSE calculation result file in ParaView.  

Generally, I used to drag the output file from MobaXterm's UI to paraview to check my results.  
However, this work doesn't work today, and I couldn't visualize my files.   

I don't know why this situation has occurred.   
Attached is a picture of the error code from Paraview. 
Please let me know how can I solve this problem and make it normally.   
![error](https://user-images.githubusercontent.com/106044260/190599037-bc68ac98-ec7b-486f-9a3b-fffe5922374d.PNG)

Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You need to use the legacy exodus reader. 
Please search on the forum, there's others who did this

Guillaume
